templates.json

```diff
@@ -458,6 +458,27 @@
                 "mainFileBreakpointLine": 12,
                 "successDebugMessage": "Hello Torizon!"
             }
+        },
+        {
+            "img": "zig-console.png",
+            "folder": "zigConsole",
+            "description": "Zig Console Application",
+            "runtime": "no-libc",
+            "language": "Zig",
+            "debugSSHPort": "2320",
```

Comment:
here we are setting "2320" but in the docker-compose.yml using the "2230".
Please change the docker-compose.yml to use "2320" too.